README.md

```diff
@@ -66,3 +77,9 @@ await conn.query(
     'update users set age=? where name=?',
     [26, 'Bob']);
 ```
+
+After making an update or addition, get number of affected rows:
+
+```dart
+print(${result.affectedRows})
```

Comment:
Thanks for the contribution.

One little thing. This syntax isn't quite right. Either add quotes or remove the `${}`

Reply:
Change made